### DISCUSSION TITLE
[x][closed][manuscript] Should we require global ids of all entities for the datasets to be valid/FAIR?

### DISCUSSION THREAD

**Discussion by sveinugu** [fairtracks/fairtracks_standard#84] [Team Posts]
Should we have different levels of validation, one level being a technical one, that the metadata contains enough info that it is searchable, but not fully FAIR? Obviosly, having to register samples/experiments/studies at central repositories is much work, but perhaps essential to calling the metadata FAIR

**Comment by sveinugu** [comment]
José María's answer: It depends. But stable ids , findable through a custom namespace registered at identifiers.org entry, are advisable.

**Comment by sveinugu** [comment]
Daniel's comments: 

* Study:
  * I don't think we can expect global identifiers for studies
* Experiment:
  * I don't think a global ID can be required in the current context.
* Track:
  * Once again, I don't see that a global ID is possible

**Comment by sveinugu** [comment]
@dzerbino 
1. Regarding track global ids. As mentioned in the meeting, my idea is that we propose to provide a global track identifier service as a delivery for a follow-up project. This service would then be registered in identifiers.org, with e.g. CURIES like: "fairtracks:12345678"
2. I agree that we cannot expect global identifiers for studies, experiments, and even samples. But would such datasets then really be considered FAIR (philosophically speaking)? At least providing the required metadata in our standard could be considered "fair" (with lowercase letters), but having access to the e.g. raw datasets as submitted to GEO or similar would be really "FAIR", hence the idea of two levels of fairness..

**Comment by dzerbino** [comment]
1. We hit a chicken and egg problem: should the attribute be required before we implement a track service?
2. I think this entire project is an exercise in compromise with reality. FAIR is reasonable when you generate data, less so when you are downstream of a bunch of people. I think we should highlight these issues in our report and move forward rather than predicate our schema on a future of milk and honey.

**Comment by sveinugu** [comment]
1. Definitely not. We can make it required at a later point, if needed.
2. Sure, agree with that. On the other hand, if metadata does not point outside of our track metadata world into some stable repository with e.g. the raw data, that should not in my opinion get an ELIXIR FAIR stamp (as we were discussing some time ago). But it should be searchable and usable through our infrastructure (lowercase 'fair'). But I agree that we can highlight this discussion on the report and continue with the lowercase 'fair' solution for now.
